### PR TITLE
RO-24 - Fixed LTI credentials handling in oauth signature validation subscriber

### PR DESCRIPTION
**Pull request by Skyld**
- Fixed `OAuthSignatureValidationSubscriber` to read LTI credentials from configuration instead of database.

### CODE REVIEW

**Review by uncleempty — APPROVED** [review]
Thanks for the quick fix! :+1: 
Just one small question, regarding changes:
should we really remove getters from `Infrastructure` entity? If yes -maybe we can go and remove this entity itself?

**Comment by Skyld** [comment]
> Thanks for the quick fix! 👍
> Just one small question, regarding changes:
> should we really remove getters from `Infrastructure` entity? If yes -maybe we can go and remove this entity itself?

Good point. Yes we can clean up, but it requires a huge effort. Purpose of this PR is to entirely clean up the infrastructure entity: https://github.com/oat-sa/simple-roster/pull/157

In order to get there what we need is:

- Merge https://github.com/oat-sa/simple-roster/pull/162 (after updating it with this fix)
- Merge https://github.com/oat-sa/simple-roster/pull/120
- Merge https://github.com/oat-sa/simple-roster/pull/157

**Comment by uncleempty** [comment]
Oki then! I fully agree that this is out of scope of underlying ticket. Thanks for a quick answer!

File: tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php [hunk 1]
```diff
@@ -72,30 +77,28 @@ public function testItReturns401IfWrongAuthentication(): void
 
         $this->kernelBrowser->request(
             'POST',
-            '/api/v1/lti/outcome? '. $queryParameters,
+            '/api/v1/lti/outcome? ' . $queryParameters,
```
Comment (andreluizmachado) [inline tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php:80]: There's a space between the question mark and querystring parameter. is this correct?

**Review by andreluizmachado — CHANGES_REQUESTED** [review]

File: tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php (hunk 1 above)
Comment (Skyld) [reply tests/Functional/Action/Lti/UpdateLtiOutcomeActionTest.php:80]: No it's not, nice catch :)

**Review by uncleempty — APPROVED** [review]

**Review by andreluizmachado — COMMENTED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by andreluizmachado — APPROVED** [review]

**Pull request merged**